docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -263,22 +264,25 @@ kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8
 
 **Note:**
 
- - For flannel to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed to `kubeadm init`.
- - flannel works on `amd64`, `arm`, `arm64` and `ppc64le`, but for it to work on an other platform than
+ - For `flannel` to work correctly, `--pod-network-cidr=10.244.0.0/16` has to be passed to `kubeadm init`.
+ - `flannel` works on `amd64`, `arm`, `arm64` and `ppc64le`, but for it to work on an other platform than
```

Comment:
>on an other platform than `amd64`

on a platform other than `amd64` ?

Reply:
Yes,  in  line #269

Reply:
...
I meant a suggestion, rephrase the sentence from "on an other platform than amd64" to "on a platform other than amd64"

Reply:
Thanks, fix it.